core/src/scene/sceneLoader.h

```diff
@@ -27,6 +28,7 @@ struct MaterialTexture;
 struct Filter;
 
 using Mixes = std::vector<YAML::Node>;
+using UniqueStyles = std::unordered_set<std::string>;
```

Comment:
`Mixes` and `UniqueStyles` seem like they should rightly be variable names, not types.